docs/UseCases.md

```diff
@@ -0,0 +1,47 @@
+## About Professor M's School for Gifted Coders
+Professor M aka Professor Marum has created an exclusive not-for-profit school for gifted coders. Coders are a subspecies of human who are born with superhuman capabilities in technology and analysis. 
+
+Professor M is gathering the best and brightest to train them to do battle against the forces of evil (specifically bad customer experiences and poor project implementations). It is hazardous and demanding work. 
+
+Professor M has deployed Sugar as a platform to help him manage both students and faculty so he can focus on teaching.
+
+##High Level Use Cases
```

Comment:
This is a really good idea.  Do we want to add high level implementation details here or link to the relevant PR/commits?  If I wanted to do implement something similar, I'd want to know what you did in Studio and how you knew how to implement the rest of the changes (related Sugar Doc or references).

Reply:
Also, we should reference this in the README